### PR TITLE
added margin top in navbar

### PR DESCRIPTION
**Pull request by ghost**
Heyy i just added the margin top in navbar.. This is my first PR thanks :)

### CODE REVIEW

File: src/components/Navbar.vue [hunk 1]
```diff
@@ -1,87 +1,70 @@
...
-    name: "navbar",
-    props: { title: String },
+  name: "navbar",
+  props: { title: String }
 };
 </script>
 
 <style scoped>
+#navbar {
+  margin-top: 5px;
```
Comment (shivam-51) [inline src/components/Navbar.vue:60]: Great work!

**Review by shivam-51 — CHANGES_REQUESTED** [review]
Hi @andikhadian Thanks for the change, Though you fulfilled the isse's requirements, you made a big mistake by changing the indentation of all the other lines. Please see because of that Navbar.vue is now +48-65 where it should have been just +3-0. Please do not change the indentation of other lines because that is not required.
Send in a new commit soon :)

**Comment by andikhadian** [comment]
Hi @shivam-51 

I am sorry about the indentation, it happened when I'm trying to save the document, and the indentation changed. Here I send my new commit. Thanks :)

**Comment by shivam-51** [comment]
> Hi @shivam-51
> 
> I am sorry about the indentation, it happened when I'm trying to save the document, and the indentation changed. Here I send my new commit. Thanks :)

@andikhadian Hi, Are you sure the indentation error has been resolved? This is happening because you must have enabled a code formatter in your IDE.
stash all your changes and open the file with some other IDE without any formatter then add the line required. Hope its clear

**Comment by andikhadian** [comment]
@shivam-51 Hi, i just re fork your repo.. and I solved the issue with another IDE.. can I PR again ?

**Comment by shivam-51** [comment]
@andikhadian You don't need to fork multiple times niether you need to clone multiple times.
Anyways, Yes you can PR  again :)

**Pull request closed without merging**